index.js

```diff
@@ -61,6 +61,14 @@ module.exports = fp(function from (fastify, opts, next) {
       }
     }
 
+    // according to https://tools.ietf.org/html/rfc2616#section-4.3
+    // fastify ignore message body when it's a GET request
+    // when proxy this request, we should reset the content-length to make it a valid http request
+    // discussion: https://github.com/fastify/fastify/issues/953
+    if (req.method === 'GET') {
+      headers['content-length'] = Buffer.byteLength(body)
```

Comment:
I would reset it to zero. GETs do not have bodies, and we should not forward them.